### PR TITLE
Delete function

### PR DESCRIPTION
**Pull request by MerleMak**

### CODE REVIEW

File: src/components/LogEntry/LogEntry.js [hunk 1]
```diff
@@ -1,26 +1,40 @@
 import styled from 'styled-components';
+import { GrTrash } from 'react-icons/gr';
+import Button from '../Button/Button';
 
-export default function LogEntry({ logEntryData }) {
+export default function LogEntry({ logEntryData, onDelete, _id }) {
+  console.log(logEntryData);
```
Comment (Zaid0307) [inline src/components/LogEntry/LogEntry.js:6]: hier ist noch eine console.log

File: src/components/LogEntryList/LogEntryList.js [hunk 2]
```diff
@@ -2,15 +2,20 @@ import LogEntry from '../LogEntry/LogEntry';
...
 // check if there are logEntries?
-export default function LogEntryList({ logEntries }) {
+export default function LogEntryList({ logEntries, onDelete }) {
   return (
     <Wrapper>
       <Header id="header">your log entries</Header>
       <EntryList role="list" aria-labelledby="header">
-        {logEntries.map((logEntryData, index) => {
+        {logEntries.map(logEntryData => {
+          console.log(logEntryData._id);
```
Comment (Zaid0307) [inline src/components/LogEntryList/LogEntryList.js:11]: für den Fall, dass du die console.logs nicht mehr brauchst

**Review by Zaid0307 — COMMENTED** [review]
Nice one 😄

File: src/components/LogEntryList/LogEntryList.js [hunk 3]
```diff
@@ -2,15 +2,19 @@ import LogEntry from '../LogEntry/LogEntry';
...
-        {logEntries.map((logEntryData, index) => {
+        {logEntries.map(logEntryData => {
           return (
-            <li key={index}>
-              <LogEntry logEntryData={logEntryData} />
+            <li key={logEntryData._id}>
+              <LogEntry
+                logEntryData={logEntryData}
+                onDelete={onDelete}
+                _id={logEntryData._id}
```
Comment (macfrei) [inline src/components/LogEntryList/LogEntryList.js:16]: You could save yourself the trouble of passing the `_id` as prop and just pass it to the `onDelete` function here.
```suggestion
                onDelete={() => onDelete(logEntryData._id)}
```
In your `LogEntry` component you'd then just write:
```js
      <Button
        type="button"
        variant="invisible"
        onClick={onDelete}
        aria-label="Delete this log entry"
      >
        <GrTrash />
      </Button>
```

**Review by macfrei — COMMENTED** [review]
Nice one, added a few comments you might want to consider 👍

File: src/App.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ import { nanoid } from 'nanoid';
 
 export default function App() {
   const [logEntries, setLogEntries] = useState([]);
+  console.log(logEntries);
```
Comment (macfrei) [inline src/App.js:10]: Llalalala console.log :P

**Pull request merged**